### ISSUE TITLE
PK violation not properly reported

### ISSUE THREAD

**Issue by arthurpaimarnold** [SAP/go-hdb#21]
When trying to bulk insert multiple rows, the driver will return error `unexpected part kind partKind(x)`, `x` being an integer. For example, using a two-table column where the first one is a primary key:
```go
stmt.Exec("0", "ABC")
stmt.Exec("1", "DEF")
stmt.Exec("2", "GHI")
stmt.Exec() // no error

stmt.Exec("0", "ABC")
stmt.Exec("1", "DEF")
stmt.Exec("2", "GHI")
stmt.Exec() // partKind error
```
The last line will produce the part kind error instead of reporting Primary Key violation. In our actual software, we're also facing the issue that the table turns out empty, even though it should contain the 3 rows inserted by the first four lines. I could not, however, reproduce this in an isolated Go source.

**Comment by d024441** [comment]
we are going to double check and provide a fix as soon as possible

**Comment by d024441** [comment]
The first issue should be fixed with the latest version 0.12.0. For the second issue we would need some more information to reproduce - please open a new ticket, if this is going to be available.

**Comment by arthurpaimarnold** [comment]
Wonderful. Thank you, Stefan!
